sdks/java/core/src/main/java/org/apache/beam/sdk/io/FileBasedSink.java

```diff
@@ -556,25 +565,35 @@ public ResourceId apply(ResourceId tempDirectory) {
      */
     @Experimental(Kind.FILESYSTEM)
     public WriteOperation(FileBasedSink<?, DestinationT, OutputT> sink, ResourceId tempDirectory) {
-      this(sink, StaticValueProvider.of(tempDirectory));
+      this(sink, StaticValueProvider.of(tempDirectory), false);
```

Comment:
Shouldn't this be "true" ? This might also explain the failure.

Reply:
The failures seem relevant as the test also performs a write before reading.

Trying to reproduce locally so I can debug.  I'm wondering if it is failing since it is trying to write to some protected dir in tests or a directory needs to be created.

I don't think that should be true there because the previous behavior was to not subdirectory, as the staticvalueprovider was used instead of the TemporaryDirectoryBuilder provider that created the subdir used by other constructor.

Reply:
Any updates ?